doc/classes/Array.xml

```diff
@@ -237,6 +237,15 @@
 				[b]Note:[/b] On large arrays, this method will be slower if the removed element is close to the beginning of the array (index 0). This is because all elements placed after the removed element have to be reindexed.
 			</description>
 		</method>
+		<method name="fill">
+			<return type="void">
+			</return>
+			<argument index="0" name="value" type="Variant">
+			</argument>
+			<description>
+				Inserts a new element at every position of the array.
```

Comment:
Sounds like this was adapted from the `insert` description, that doesn't sound clear to me. I'd suggest instead:
```suggestion
				Initializes all elements in the array with the given value. This can typically be used together with [method resize] to create an array with a given size and initialized elements:
				[codeblock]
				var arr = []
				arr.resize(10)
				arr.fill(0)  # Initialize the 10 elements to 0.
				[/codeblock]
```

Reply:
Indeed, it was adapted from `insert`. I will update the descriptions ASAP, but I might change the word "initialize" to something else. I don't know if it's just me, but it gives me the ideia of something that you do when creating the array, but you can call **array.fill()** whenever (you could use it to "reset" an array or something).

Reply:
Yes it could be "Sets all elements in the array to the given value." or "Assigns the given value to all elements in the array."

Reply:
I've updated the documentation. Thanks for the help and suggestions!